testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_replication_eventing.py

```diff
@@ -552,15 +552,4 @@ def test_push_replication_for_20mb_doc(params_from_base_test_setup, attachment_g
     repl = replicator.create(repl_config)
     repl_change_listener = replicator.addReplicatorEventChangeListener(repl)
 
-    # 2. Starting Replication and waiting for it finish
```

Comment:
This code not required for Lithium to check for replication even changes? Could you explain?